prices.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import csv
+import os
+import time
+import calculations
+
+
+def generate_general_statistic(data, filename):
+    item_id_index = 0
+    store_id_index = 1
+    price_index = 2
+
+
+    with open('Data_Output/' + filename + '_general_statistic.csv', 'w', newline='') as file:
```

Comment:
Use `os.path.join` or `pathlib` to concatenate paths. Current code will fail on some OSes